src/plugin/error.py

```diff
@@ -1,12 +1,15 @@
 from spaceone.core.error import *
 
 
-class ERROR_REQUIRED_FIELDS(ERROR_BASE):
-    _message = 'Required field is missing(field= {field})'
+class ERROR_INVALID_WEBHOOK_TYPE(ERROR_BASE):
```

Comment:
```suggestion
class ERROR_INVALID_WEBHOOK_TYPE(ERROR_INVALID):
```
I think it would be better use ` ERROR_INVALID` for application error not `ERROR_BASE`